Sources/DatadogObjc/RUMMonitor+objc.swift

```diff
@@ -257,19 +257,27 @@ public class DDRUMMonitor: NSObject {
     public func stopResourceLoading(
         resourceKey: String,
         response: URLResponse,
+        size: NSNumber?,
         attributes: [String: Any]
     ) {
-        swiftRUMMonitor.stopResourceLoading(resourceKey: resourceKey, response: response, attributes: castAttributesToSwift(attributes))
+        swiftRUMMonitor.stopResourceLoading(resourceKey: resourceKey, response: response, size: size?.int64Value, attributes: castAttributesToSwift(attributes))
     }
 
     @objc
     public func stopResourceLoading(
         resourceKey: String,
-        statusCode: Int,
+        statusCode: NSNumber?,
```

Comment:
why do we need this change?

Reply:
`statusCode` is an optional parameter to our `RUMMonitor` API and because Objc doesn't have in-build optional type, this must be mapped to a null-able reference type (`NSNumber`).